doc/rst/usingchapel/chplenv.rst

```diff
@@ -660,6 +660,7 @@ CHPL_HWLOC
        ======== ==============================================================
        none     do not build hwloc support into the Chapel runtime
        bundled  use the hwloc distribution bundled with Chapel in third-party
+       system   use a system install of hwloc (requires version 2.1+)
```

Comment:
The sentence above this table also needs to be updated to reflect the fact there are now 3 options.